frontend/src/scenes/project/Settings/AutocaptureSettings.tsx

```diff
@@ -17,19 +20,40 @@ export function AutocaptureSettings(): JSX.Element {
                 Automagically capture frontend interactions like pageviews, clicks, and more when using our JavaScript
                 or React Native libraries.
             </p>
-            <LemonSwitch
-                id="posthog-autocapture-switch"
-                onChange={(checked) => {
-                    updateCurrentTeam({
-                        autocapture_opt_out: !checked,
-                    })
-                    reportIngestionAutocaptureToggled(!checked)
-                }}
-                checked={!currentTeam?.autocapture_opt_out}
-                disabled={userLoading}
-                label="Enable Autocapture"
-                bordered
-            />
+            <div className={'flex flex-col space-y-2'}>
```

Comment:
flex and flex-col are probably not needed here. If they are then `gap-2` is preferable for flexed content

```suggestion
            <div className="space-y-2">
```